tests/commands/info.rs

```diff
@@ -0,0 +1,41 @@
+use assert_cmd::Command;
+use mockito::mock;
+use predicates::prelude::*;
+use predicates::str::contains;
+
+use crate::common;
+
+#[test]
+fn info_works_when_all_required_env_are_present() {
```

Comment:
I think the convention is to prefix these with `test_`?

Reply:
I didn't find anything about this in Rust Book, and I like how the output looks now:

![image](https://user-images.githubusercontent.com/1523305/86125397-e07bc280-badc-11ea-96bf-31cf97e82e7d.png)